sdcm/nemesis.py

```diff
@@ -1216,6 +1216,7 @@ def disrupt(self):
 class StopWaitStartMonkey(Nemesis):
 
     disruptive = True
+    run_with_gemini = False
```

Comment:
Why?
This is disruptive nemesis that currently isn't configured to run with Gemini, but will be after non-disruptive will work.

Reply:
please remove.